data/schemas/keyboard.jsonschema

```diff
@@ -133,6 +133,14 @@
                 },
                 "macro_beep": {"type": "boolean"},
                 "pins": {"$ref": "qmk.definitions.v1#/mcu_pin_array"},
+                "enable_pin": {
```

Comment:
I'm not really happy with the name, but audio.enable just doesn't seem right.

Reply:
`power_control` perhaps?

Reply:
Yeah, that's a much better name.  Updated names.